.github/vendor-templates/microsoft/configure_linux/action.yml

```
@@ -12,4 +12,9 @@ runs:
       - name: Configure xvfb
         shell: bash
         run: |
-          Xvfb :1 -screen 0 1024x768x16 &
\ No newline at end of file
+          Xvfb :1 -screen 0 1024x768x16 &
+
+      - name: configure Python3
+        shell: bash
+        run: |
+          sudo ln -s /usr/bin/python3 /usr/bin/python
```

NIT. As a good practice it should go to `/usr/local/bin/python`.

```suggestion
          sudo ln -s /usr/bin/python3 /usr/local/bin/python
```

---

Hopefully this will still work 😉 !

---

Do we want to test this first before merging?